### PR TITLE
port to 2.0

### PR DESCRIPTION
**Pull request by notwaldorf**
Straight forward hybrid port.

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
 <body>
   <script>
     WCT.loadSuites([
-      'basic.html',
-      'events.html',
+      'basic.html?wc-shadydom=true&wc-register=true',
+      'events.html?wc-shadydom=true&wc-register=true',
```
Comment (e111077) [inline test/index.html]: wc-ce

File: test/basic.html [hunk 2]
```diff
@@ -73,9 +73,15 @@
   <script>
     suite('basic', function() {
 
-      test('id generated on creation', function() {
+      test('id generated on creation', function(done) {
         var e = document.createElement('iron-label');
-        assert(e.id, 'iron-label-0');
+        document.body.appendChild(e);
+        Polymer.Base.async(function() {
```
Comment (e111077) [inline test/basic.html]: why is this async now?

Comment (notwaldorf) [inline test/basic.html]: unclear; maybe old polymer quirks. removing!

**Comment by notwaldorf** [comment]
@e111077 done fixed!

**Pull request merged**